events/templates/events/eventslist.html

```diff
@@ -27,106 +27,149 @@
                                             details
                                         </button>
-                                    </small>
-                                </a>
+                                    </div>
+                                </div>
                             </div>
                             <div class="collapse" id="list-{{event.id}}" aria-labelledby="{{event.id}}_heading" data-parent="#eventslist">
-                                <div class="card-body p-2">
-                                    {{event.name}}
-                
-                                    <div class="d-flex w-100 justify-content-between">
-                                        <small>
-                                            {{event.location.name}},
-                                            {{event.location.street}},
-                                            {{event.location.place}}
-                                        </small>
+                                <div class="card-body p-0 mt-0">
+                                    <ul class="list-group list-group-flush">
+                                        {% if user.is_authenticated %}
+                                        <li class="list-group-item pl-2 pt-0 pr-2 pb-1 mb-0 mt-0">
+                                            <div class="row">
+                                                <div class="col">
+                                                    <small>who's in? 👍</small>
+                                                </div>
+                                            </div>
+                                            <!-- <hr style="margin:1px; padding:0px"> -->
+                                            <div class="row float-left pl-0 ml-0" style="line-height:1;"> 
+                                                {% for p in participants %}
+                                                {% if p.part.choice == "y" %}
+                                                <div class="col m-0 p-0" >
+                                                    <img src="{{p.person.profile.image_url}}" 
+                                                    class="img m-0 p-0" 
+                                                    alt="{{p.person.first_name}}" 
+                                                    style="object-fit:cover;
+                                                            width:70px;
+                                                            height:70px;
+                                                            border: solid 0px #CCC"/>
+                                                    <br class="p-0 m-0">
+                                                    <small class="pt-0 mt-0">{{p.person.first_name}}</small>
```

Comment:
es fällt auf, dass dieser block, der das foto des/der Teilnehmer*in mit Namen anzeigt in dieser Form mindestens 4 Mal auftaucht:
1. Im yes block
2. Im no block
3. Im maybe block
4. In der Sidebar

Um weniger Code zu duplizieren, könnte man das Erzeugen des `div`s aus einem participant in eine eigene Funktion auslagern:
https://stackoverflow.com/a/1333277/10009545